### PR TITLE
fix(core): Remove shared credentials data from console messages

### PR DESCRIPTION
**Pull request by cstuncsik**

### CODE REVIEW

File: packages/cli/src/WorkflowExecuteAdditionalData.ts [hunk 1]
```diff
@@ -1042,14 +1050,45 @@ async function executeWorkflow(
 	};
 }
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export function sendMessageToUI(source: string, messages: any[]) {
+/* eslint-disable @typescript-eslint/no-explicit-any, @typescript-eslint/no-unsafe-return */
+export async function sendMessageToUI(source: string, messages: any[]): Promise<void> {
```
Comment (netroy) [inline packages/cli/src/WorkflowExecuteAdditionalData.ts:1054]: We should not be disabling linting for entire code blocks. we have enough tech-debt already, this only increases the tech-debt.

**Comment by netroy** [comment]
Is there a ticket or a slack conversation for more details ?

File: packages/cli/src/WorkflowExecuteAdditionalData.ts (hunk 1 above)
Comment (cstuncsik) [reply packages/cli/src/WorkflowExecuteAdditionalData.ts]: This is just WIP and also a wrong approach, just reverted it

**Comment by cstuncsik** [comment]
> Is there a ticket or a slack conversation for more details ?

@netroy unfortunately not much except the [linear ticket](https://linear.app/n8n/issue/N8N-5927/dont-leak-http-creds-in-the-browser-console). I had a conversation with @janober about how to better approach this and we agreed that not just shared but all credentials should be removed/obfuscated

File: packages/cli/src/WorkflowExecuteAdditionalData.ts (hunk 1 above)
Comment (netroy) [reply packages/cli/src/WorkflowExecuteAdditionalData.ts]: ah. can you please then mark the PR as `in draft` ?

**Pull request closed without merging**